scripts/nebula.service

```diff
@@ -234,6 +234,9 @@ function status_daemon {
             port=${GREEN}${port}${NC}
         else
             port=${BLINK}${RED}${port}${NC}
+            if [[$daemon_name == nebula-storaged]]; then
```

Comment:
How do you let others know ADD  HOSTS is a command?

Reply:
Maybe a link to faq in docs?

Reply:
Maybe a good idea

Reply:
> Maybe a link to faq in docs?

I don‘t think so, it’s a little werid.